src/cli.js

```diff
@@ -35,6 +35,12 @@ const yargv = yargs
   .boolean('commit')
   .default('files', ['README.md'])
   .default('contributorsPerLine', 7)
+  .option('contributorsSortAlphabetically', {
```

Comment:
Wouldn't only having a shorter option name say `alphabeticalSort` be concise and clear or would that only be for me?

Reply:
I picked this name to be consistent with `contributorsPerLine`.